data-model/src/main/scala/za/co/absa/enceladus/model/conformanceRule/package.scala

```diff
@@ -40,6 +40,11 @@ package object conformanceRule {
     val controlCheckpoint: Boolean
 
     def withUpdatedOrder(newOrder: Int): ConformanceRule
+
+    def hasEntityConnected: Boolean = false
+    def maybeConnectedEntityName: Option[String] = None
```

Comment:
I would prefer not to name it as maybe.... Why not just like connectedEntityName?

Reply:
I think maybe signifies that it is an option and I think I got it on CQC.

Reply:
My view is that a type itself is also a part of the documentation and this style reminds me of Hungarian notation. @dk1844 and @benedeki , what are your opinions on this?

Reply:
I agree with the guys that just simple `connectedEntityName: Option[String]` would suffice.

A last resort scenario could be `optConnectedEntityName`, only for cases when you have an optional variant and then a defined variant, and you somehow need both to work together. Further in code, you work with validations that are actually of type `Future[Validation]`, but you do not normally include the `future` word in the field/method name either.

Reply:
Fixed